libraries/SoaringController/MatrixMathExample.pde

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
This file isn't part of the build, just an example of how to use MatrixMath routines. Maintaining it seems more trouble than it's worth though, so I removed it in PR#5737.